libsql-server/src/http/admin/stats.rs

```diff
@@ -140,10 +140,12 @@ pub(super) async fn handle_stats<C>(
     State(app_state): State<Arc<AppState<C>>>,
     Path(namespace): Path<String>,
 ) -> crate::Result<Json<StatsResponse>> {
+    dbg!();
```

Comment:
I am reviewing commit by commit, so not sure if these got removed but wanted to note it.